pages/_admin/index.tsx

```diff
@@ -1,29 +1,36 @@
-import { useRouter } from 'next/router'
-import { Box,
-  Button,
-  Container,
-  Heading,
-  Stack,
-  Text } from '@chakra-ui/react'
+import { Box, Button, Container, Heading, Stack, Text } from '@chakra-ui/react'
 import { useSession, getSession } from 'next-auth/client'
+import Link from 'next/link'
 
 import prisma from '../../lib/prisma'
 import Nav from './nav'
 
-function Story({ id, title, content, name, postal, ...rest}) {
+function Story({ id, title, content, name, postal, email, phone, twitter, ...rest }) {
   return (
     <Box mt={2} p={5} shadow="md" borderWidth="1px" {...rest}>
       <Heading fontSize="xl">{title}</Heading>
       <Text mt={4}>{content}</Text>
-      <Text mt={4}>{name} from {postal}</Text>
+      <Text mt={4} mb={4}>
+        {name || `Anonymous`} from {postal}
+      </Text>
+      <Text mt={4} mb={4}>
+        Contact: {email || twitter || phone ? `Yes` : 'No'}
+      </Text>
+      <Text mt={4} mb={4}>
+        ID: <Link href={`/story/${id}`}>{id}</Link>
+      </Text>
       <Stack direction="row" spacing={4} align="center">
         <Button
-          colorScheme="blue"
+          colorScheme="red"
           variant="outline"
           type="button"
           data-id={id}
           data-type="deleted"
-          onClick={(e) => {if(window.confirm('Are you sure you want to delete this?')){updateStory(e)}}}
+          onClick={(e) => {
+            if (window.confirm('Are you sure you want to delete this?')) {
+              updateStory(e)
+            }
+          }}
         >
```

Comment:
Show mods if a submission has consented to contact, and include the story ID (and link to what would be the live link if approved)